options.html

```diff
@@ -62,12 +62,22 @@ <h1>Settings</h1>
 			</label>
 		</p>
 
+		<p>
+			<label>
+				<textarea id="allowlist" value="" cols="80">
+				</textarea>
+				<br>
+				Blacklist urls, put websites into Google Container (for example third party SSO)<br>
```

Comment:
what `does` Blacklist here mean?

Reply:
Blacklist means a list of host that you want to use Google Container (outside from google websites). Is an input of @hackerncoder .

Reply:
What about `Include additional urls in Google Container` and as explanation below `Means open additional urls in Google container, e.g. for third party SSO`

Reply:
I agree about the headline. For the explanation, my proposal would be `(Means use Google Container on these additional urls, e.g. for third party SSO or "Log in with Google". Use one url per line.)`. 

So as to standardize all the preferences, we can change as well the last option (Whitelisted urls). Until I checked the code, I hadn't understood who it works. Maybe that text are more concrete:

> headline: `Exclude Google urls in Google Container`
> explanation: `(Means don't use Google Container on these Google urls. Use one url per line.)`

Are they ok? I can commit all of them :)

Reply:
The opposite of whitelist? I couldn't find a better word. I thought that it at least gave more information to the user compared to `Allowlist`

Reply:
I've just made a commit with my proposal of allowlist and whitelist explanations text, trying to include the @Perflyst opinion, waiting to know if there are ok. I mark as resolve and we can open another review if it's necessary

Reply:
Sorry for the late reply, LGTM :)